### PR TITLE
test

### PR DESCRIPTION
**Pull request by newcoder171**

### CODE REVIEW

**Review by inhaq — CHANGES_REQUESTED** [review]
Hi team, `tictactoe` tests are resulting an error. Kindly fix. Here are suggestions to improve testing for this project.
 - Unit testing should be done for individual component/function. Here you're just running the game using rspec which isn't making any sense
 - You should provide input and expect some output from that function(which you're testing)
 - You can check winning combination for game logic etc
 - Also kindly put `.idea` in `.gitignore` file to avoid pushing unnecessary files
 
Once fixed and updated with new unit tests, kindly resubmit for review.

File: spec/tictactoe_spec.rb [hunk 1]
```diff
@@ -1,22 +1,55 @@
 require 'rspec'
+require_relative '../bin/tictactoe'
 
-# Our RSpec test is here
-require_relative '../lib/player'
 
+describe 'TicTacToe' do
+  let!(:player1) { Player.new('Tester1', 'o') }
```
Comment (inhaq) [inline spec/tictactoe_spec.rb]: why to use `let!` instead of `let`

File: spec/tictactoe_spec.rb [hunk 2]
```diff
@@ -1,22 +1,55 @@
...
+    expect(game.board).to eq [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  let!(:switch) { game.switch}
+  it 'switches the players' do    
+    expect(game.players[0]).to eq switch[1]
+  end
+
+  
+  it 'should return true for winnig combinations' do
```
Comment (inhaq) [inline spec/tictactoe_spec.rb]: You should cover most of the winning and tie combinations

**Review by inhaq — CHANGES_REQUESTED** [review]
Hi team, good effort. You're one step away from completing this project. Kindly follow the comment and update your code. Kindly submit for review once updated.

File: spec/tictactoe_spec.rb (hunk 1 above)
Comment (newcoder171) [reply spec/tictactoe_spec.rb]: I was having issues running the test at first, so I was trying out anything I felt could be the solution

File: spec/tictactoe_spec.rb (hunk 2 above)
Comment (newcoder171) [reply spec/tictactoe_spec.rb]: okay

File: spec/tictactoe_spec.rb (hunk 1 above)
Comment (inhaq) [reply spec/tictactoe_spec.rb]: You've done an amazing job and believe me, learning rspec will be a huge help to you in future. [Here](https://stackoverflow.com/a/10177103/6730682) you can see the difference between `let` and `let!`.

**Review by inhaq — APPROVED** [review]
### 👍🏻Good job
Remember that in unit tests, you'll need to cover most of the use cases and code coverage. 
You can go ahead and submit your project completion form. Good luck with next projects 🎉

**Pull request merged**